arch/x86/core/acpi.c

```diff
@@ -6,7 +6,7 @@
 #include <arch/x86/acpi.h>
 
 static struct acpi_rsdp *rsdp;
-bool is_rdsp_searched = false;
+bool is_rsdp_searched;
```

Comment:
Shouldn’t this be static? Same with is_dma_searched.

Reply:
Added `static`. I was just correcting the typo and didn't realize this could be set `static`.